Description: Fix library name in test_IkFast by removing d prefix
 The code sets library d prefix if not NDEBUG is present. This conflicts
 with the debian build infrastructure.
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Last-Update: 2019-01-24
Forwarded: No

diff --git a/unittests/unit/test_IkFast.cpp b/unittests/unit/test_IkFast.cpp
index e926897..7b35dfa 100644
--- a/unittests/unit/test_IkFast.cpp
+++ b/unittests/unit/test_IkFast.cpp
@@ -74,9 +74,6 @@ TEST(IkFast, LoadWamArmIk)
   ik->setTarget(targetFrame);
   ik->setHierarchyLevel(1);
   std::string libName = "libGeneratedWamIkFast";
-#if (DART_OS_LINUX || DART_OS_MACOS) && !NDEBUG
-  libName += "d";
-#endif
 #if DART_OS_LINUX
   libName += ".so";
 #elif DART_OS_MACOS
